media-sound/carla/carla-9999-r1.ebuild

```diff
@@ -23,7 +22,7 @@ fi
 LICENSE="GPL-2 LGPL-3"
 SLOT="0"
 
-IUSE="alsa gtk gtk2 opengl osc -pulseaudio rdf sf2 sndfile X"
+IUSE="alsa gtk gtk2 opengl osc pulseaudio rdf sf2 sndfile X"
```

Comment:
I guess this is removed because of some `pkgcheck` output? Are there any docs that explain why it's flagging this?

Reply:
This is from pkgcheck output. Unfortunately, I couldn't find detailed documentation on this issue, yet.
However, it is one of the red checks (aka. “critical”) and thus one which shouldn't be ignored if we don't have a particular reason to do so.

My thinking why this IUSE configuration was flagged is probably because of how IUSE defaults are processed.
According to the [devmanual](https://devmanual.gentoo.org/general-concepts/use-flags/index.html#iuse-defaults) adding a `-` before a USE flag is essential a NOOP as the setting is always overriden by the profile make.defaults, package.use or make.conf if the respective USE flag is set. If it is not set there, the USE flag is disabled anyway, so it really shouldn't matter if `-` is specified in IUSE or not. For the sake of not confusing developers (and preventing issues in case Portage decides how to interpret such a `-` in the future), omitting the `-` is a sensible choice.

Reply:
Looks like there is no further opposition to my explanation, so I consider this resolved.